templates/commands/render/action.go

```diff
@@ -141,6 +141,29 @@ func templateAndCompileRegexes(regexes []model.String, scope *common.Scope) ([]*
 	return compiled, merr
 }
 
+// processPaths processes a list of input String paths for go templating, relative paths,
+// OS-specific slashes, and (soon) file globbing.
+func processPaths(paths []model.String, sp *stepParams) ([]model.String, error) {
```

Comment:
Maybe just pass the sp.scope here instead of the entire sp?

Reply:
Changed to only pass scope.